src/boot/store.js

```diff
@@ -276,6 +276,11 @@ const migrations: {| [string]: (GlobalState) => GlobalState |} = {
     };
   },
 
+  '27': state => ({
+    ...state,
+    muted_users: Immutable.Map({}),
+  }),
```

Comment:
As you observed in the chat thread, this isn't needed when adding a new key.

Reply:
Removed.